### PR TITLE
docs: update tutorial collection

### PR DESCRIPTION
**Pull request by maximilianwerk**
Updated the tutorial section to be in sync with the currently running examples. Hope I got the buzzwords right.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,27 +275,51 @@ Search South Park scripts and practice with Flows and Pods
...
+<h4><a href="https://github.com/jina-ai/examples/tree/master/multires-lyrics-search">Use Chunk to search Lyrics</a></h4>
+Split documents in order to search on a finegrained level
+</td>
+</tr>
+
+<tr>
+<td><h3>🕊</h3></td>
+<td>
+<h4><a href="https://github.com/jina-ai/examples/tree/master/cross-modal-search">Mix and Match images and captions</a></h4>
+Search cross model to get images from captions and vice versa
```
Comment (JoanFM) [inline README.md:303]: typo in model (it is cross modal)

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by alexcg1** [comment]
Somehow the table got dropped at the end of the README :(

**Comment by maximilianwerk** [comment]
> Somehow the table got dropped at the end of the README :(

That was a missing table tag.

File: README.md [hunk 2]
```diff
@@ -275,47 +275,71 @@ Search South Park scripts and practice with Flows and Pods
...
 <td><h3>🚀</h3></td>
 <td>
 <h4><a href="https://github.com/jina-ai/examples/tree/master/pokedex-with-bit">Google's Big Transfer Model in (Poké-)Production</a></h4>
 Search Pokemon with state-of-the-art visual representation
 </td>
 </tr>
+ -->
 </table>
-
 ## Documentation
```
Comment (hanxiao) [inline README.md:332]:
```suggestion

## Documentation
```

**Review by hanxiao — APPROVED** [review]

**Pull request merged**